app/src/main/java/com/nilhcem/blenamebadge/ui/fragments/MainTextDrawableFragment.kt

```diff
@@ -61,71 +60,8 @@ class MainTextDrawableFragment : BaseFragment() {
         }
     }
 
-    override fun getSendData(): DataToSend {
```

Comment:
It is removed and copied at the bottom. Please fix this

Reply:
Fixed